gui/create-item.lua

```diff
@@ -470,7 +482,7 @@ function hackWish(unit)
       if itemtype ~= df.item_type.CORPSEPIECE and itemtype ~= df.item_type.CORPSE then
        createItem({mattype,matindex},{itemtype,itemsubtype},quality,unit,description,1)
       else
-       createCorpsePiece(unit,bodypart,partlayerID-1,mattype,matindex,corpsepieceGeneric,quality)
+       createCorpsePiece(unit,bodypart,partlayerID-2,mattype,matindex,corpsepieceGeneric,quality)
```

Comment:
i get nervous when I see a `-2` here from the issues we had with it before. we're sure it's correct now?

Reply:
there's an extra entry in the part layer prompt now so i have to offset it by two, plus it's always set so initializing it to -1 won't do anything. i _could_ do the same thing i did with generic body parts, and set a flag when the user chooses the entry in the tissue layer selection screen (which is "whole") but i feel like i want to avoid variable bloat.

Reply:
or have we been adding variables all over the place already?

Reply:
no, this is fine. as long as you have tested the various code paths that lead here.

Reply:
tested corpsepiece (single layer), corpsepiece (generic), corpsepiece (whole), and corpse, which don't seem to have any problems.